llvm/lib/Target/AIE/AIE2TargetMachine.cpp

```diff
@@ -300,3 +300,10 @@ void AIE2TargetMachine::registerPassBuilderCallbacks(
     });
   }
 }
+
+bool AIE2TargetMachine::isNoopAddrSpaceCast(unsigned SrcAS,
```

Comment:
how about adding this to `AIEBaseTargetMachine`?

Reply:
Done